packages/blitz/src/render.rs

```diff
@@ -196,16 +202,53 @@ where
     }
 
     pub fn mouse_move(&mut self, x: f32, y: f32) -> bool {
+        let RenderState::Active(state) = &self.render_state else {
+            return false;
+        };
+
+        let x = x / state.viewport.zoom();
+        let y = y / state.viewport.zoom();
+
+        // println!("Mouse move: ({}, {})", x, y);
+        // println!("Unscaled: ({}, {})",);
+        self.mouse_pos = (x, y);
         let old_id = self.hover_node_id;
         self.hover_node_id = self.dom.as_ref().hit(x, y);
         if old_id != self.hover_node_id {
-            // println!("Hovered node: {:?}", self.hover_node_id);
-            self.devtools.highlight_hover
+            true
+            // self.devtools.highlight_hover
         } else {
             false
         }
     }
 
+    pub fn get_cursor(&self) -> Option<CursorKind> {
+        // todo: cache this on the node itself
+        let node = &self.dom.as_ref().tree()[self.hover_node_id.unwrap()];
```

Comment:
I think we want to set this in the Stylo `ElementData`. That (+ some cache invalidation) ought to make hover styles work I think.